packages/bitcore-wallet-client/test/api.test.js

```diff
@@ -6526,6 +6526,37 @@ describe('client API', function() {
         });
       });
 
+      it('should not fail to gain access to eth wallet with unknown tokens addresses from mnemonic (Case 3)', done => {
+        helpers.createAndJoinWallet(clients, keys, 1, 1, { coin: 'eth' }, () => {
+          var words = keys[0].get(null, true).mnemonic;
+          var walletName = clients[0].credentials.walletName;
+          var copayerName = clients[0].credentials.copayerName;
+
+          clients[0].savePreferences({ tokenAddresses: ['0x9da9bc12b19b22d7c55798f722a1b6747ae9a710'] }, err => {
+            should.not.exist(err);
+              Client.serverAssistedImport(
+              { words },
+              {
+                clientFactory: () => {
+                  return helpers.newClient(app);
+                }
+              },
+              (err, k, c) => {
+                // the eth wallet + 1 unknown token addresses on preferences.
+                c.length.should.equal(1);
```

Comment:
Should this be 2? The comment above makes me think it should be 2, but the oneInch API doesn't have a match for the tokenAddresses passed in preferences.

Reply:
No, one is ok. I'm just testing that if there is an address it does not exist in the 1inch api retrieves the eth wallet without issues